acceptance-tests/runner/scripts/download.sh

```diff
@@ -31,15 +31,47 @@ download() {
         echo "curl or wget must be installed."
         exit 1
     fi
-    
+
     if [ $? != 0 ]
       then
         popd
         echo " ************************************"
         echo " **** ${DOWNTO} download failed."
         echo " ************************************"
         exit 1
-    fi    
+    fi
 
      popd
 }
+
+download-jenkins() {
+    pushd runner
+    if [ ! -d bin ]; then
+        mkdir bin
+    fi
+    JENKINS_VERSION=$1
+    DOWNTO=$2
+    if [ -f "$DOWNTO" ]; then
+        echo ""
+        echo "***** ${DOWNTO} already downloaded. Skipping download."
+        echo ""
+        popd
+        return
+    fi
+
+    echo ""
+    echo "Downloading Jenkins ${JENKINS_VERSION} WAR to ${DOWNTO} ..."
+    echo ""
+
+    mvn -B dependency:copy -Dartifact="org.jenkins-ci.main:jenkins-war:${JENKINS_VERSION}:war" && cp "target/dependency/jenkins-war-${JENKINS_VERSION}.war" "${DOWNTO}"
```

Comment:
this will not work :) because per default Maven knows only Centra repo and jenkins artifact are not in central.
you can use a fake pom containing repo-ci as repository or run this download in the `ath` which have a pom with repo-ci configured.